src/corehost/cli/hostpolicy/args.cpp

```diff
@@ -24,7 +24,7 @@ arguments_t::arguments_t()
  *  o %DOTNET_SHARED_STORE% -- multiple delimited paths
  *  o dotnet.exe relative shared store\<arch>\<tfm>
  *  o Global location
- *      Windows: C:\Program Files (x86) or
+ *      Windows: C:\\Program Files (x86) or
```

Comment:
```suggestion
 *      Windows: global default location (Program Files) or globally registered location (registry) + store\<arch>\<tfm>
```

Reply:
Can you please change the comments as suggested - the old ones are basically wrong...

Reply:
Fixed. Thanks!